migrate/src/main/scala/migrate/Main.scala

```diff
@@ -45,8 +45,14 @@ object Main {
     } yield result.content
   }
 
-  private def setupScala3Compiler(classpath: Classpath, classDirectory: AbsolutePath): Try[Scala3Compiler] = {
-    val scala3CompilerArgs = Array(
+  private def setupScala3Compiler(
+    classpath: Classpath,
+    classDirectory: AbsolutePath,
+    scala3CompilerOptions: Seq[String]
+  ): Try[Scala3Compiler] = {
+    // TODO: https://github.com/mlachkar/scala-migrat3/issues/24
```

Comment:
One change here

Reply:
Actually your temporary solution is a good one:
```
val modified = scala3CompilerOptions.filterNot(_ == "-Ysemanticdb")
```
I think the `-Ysemanticdb` option is never useful so why do we need to care :)